### DISCUSSION TITLE
New .spec

### DISCUSSION THREAD

**Discussion by rexy74** [phaag/nfdump#483] [nfdump-1.7]
Hi.
Thx for your job.
Here is the latest version of the .spec file replacing the
 old one (/extra/nfdump.spec) for nfdump-1.7.3

[nfdump-1.tar.gz](https://github.com/phaag/nfdump/files/13467064/nfdump-1.tar.gz)

**Comment by phaag** [comment]
Cool!! - many thanks!!
